pkg/controllers/nodeclaim/disruption/suite_test.go

```diff
@@ -113,6 +113,7 @@ var _ = Describe("Disruption", func() {
 		Expect(nodeClaim.StatusConditions().Get(v1beta1.ConditionTypeEmpty).IsTrue()).To(BeTrue())
 	})
 	It("should remove multiple disruption conditions simultaneously", func() {
+		cp.Drifted = ""
```

Comment:
I'm wondering if this was an intentional "feature" of the test, to verify that we could remove multiple disruption conditions **without** removing all conditions? Or was this change related to your other changes and I'm not realizing it?